pkgs/applications/networking/cluster/terraform/default.nix

```diff
@@ -48,8 +49,21 @@ in rec {
     sha256 = "0ibgpcpvz0bmn3cw60nzsabsrxrbmmym1hv7fx6zmjxiwd68w5gb";
   };
 
-  terraform_0_9_1 = generic {
-    version = "0.9.1";
-    sha256 = "081p6dlvkg9mgaz49ichxzlk1ks0rxa7nvilaq8jj1gq3jvylqnh";
+  terraform_0_9_2 = generic {
+    version = "0.9.2";
+    sha256 = "1yj5x1d10028fm3v3gjyjdn128ps0as345hr50y8x3vn86n70lxl";
+
+    patches = [
+      (fetchurl {
```

Comment:
Do you mean extract it into a function? The only `fetchPatch` I could find in nixpkgs was at pkgs/applications/networking/irc/bip/default.nix